Source/Core/Core/PowerPC/Expression.cpp

```diff
@@ -102,7 +103,35 @@ static double CastFunc(expr_func* f, vec_expr_t* args, void* c)
   return Common::BitCast<T>(static_cast<U>(expr_eval(&vec_nth(args, 0))));
 }
 
-static std::array<expr_func, 21> g_expr_funcs{{
+static double CallstackFunc(expr_func* f, vec_expr_t* args, void* c)
+{
+  if (vec_len(args) != 1)
+    return 0;
+
+  std::vector<Dolphin_Debugger::CallstackEntry> stack;
+  bool success = Dolphin_Debugger::GetCallstack(stack);
+  if (!success)
+    return 0;
+
+  double num = expr_eval(&vec_nth(args, 0));
+  if (!std::isnan(num))
+  {
+    u32 address = static_cast<u32>(num);
+    return std::any_of(stack.begin(), stack.end(),
+                       [address](const auto& s) { return s.vAddress == address; });
+  }
+
+  const char* cstr = expr_get_str(&vec_nth(args, 0));
+  if (cstr != NULL)
```

Comment:
Missed this before... nullptr (C++) is preferred to NULL (C) in Dolphin.

Reply:
Should expr_get_str still return NULL when invalid?

Reply:
Yes, expr.h should still use NULL because C isn't getting the nullptr until C23.